tests/framework/installer/ceph_installer.go

```diff
@@ -570,7 +570,7 @@ func (h *CephInstaller) UninstallRookFromMultipleNS(systemNamespace string, name
 		}
 
 		roles := h.Manifests.GetClusterRoles(namespace, systemNamespace)
-		_, err = h.k8shelper.KubectlWithStdin(roles, deleteFromStdinArgs...)
+		_, _ = h.k8shelper.KubectlWithStdin(roles, deleteFromStdinArgs...)
```

Comment:
ditto do not silence the error, handle it

Reply:
Done.